README.md

```diff
@@ -678,6 +688,8 @@ The `MathpixMarkdown` React element accepts the following props:
 | `showPageBreaks` | boolean;*`false`*            | Hidden tags will be shown in html like page break                                                                      |
 | `centerImages`   | boolean;*`true`*             | Center align images by default                                                                                         |
 | `centerTables`   | boolean;*`true`*             | Center align tables by default                                                                                         |
+| `validateLink`   | function;*`null`*            | The function `(url: string) => void` to override md link validator                                                     |
+| `enableCodeBlockRuleToLatexCommands`| boolean;*`false`* | By default for latex commands is to ignore rule `Code blocks` are normally indented four spaces or one tab.    |
```

Comment:
```suggestion
| `enableCodeBlockRuleForLatexCommands`| boolean;*`false`* | By default, if latex commands are indented (4 spaces / 1 tab) they do not become `Code Blocks`. |
```

would this be a better param name @OlgaRedozubova ?

Reply:
> would this be a better param name @OlgaRedozubova ?

Thanks. I'll rename it.